cmd/modern/root/install/mssql-base.go

```diff
@@ -444,31 +450,48 @@ CHECK_POLICY=OFF`
 	}
 }
 
+func getDbNameIfExists(usingDbUrl string) string {
+	if strings.Contains(usingDbUrl, ",") {
+		dbToken := strings.Split(usingDbUrl, ",")
+		if len(dbToken) > 1 {
+			return dbToken[1]
+		}
+	}
+	return ""
+}
+
 func (c *MssqlBase) downloadAndRestoreDb(
 	controller *container.Controller,
 	containerId string,
 	userName string,
 ) {
 	output := c.Cmd.Output()
+	databaseName := getDbNameIfExists(c.usingDatabaseUrl)
+	databaseUrl := c.usingDatabaseUrl
+	if databaseName != "" {
+		databaseUrl = strings.Split(c.usingDatabaseUrl, ",")[0]
```

Comment:
Don't use Split, just take everything after the first comma

Reply:
file names, folder names, and database names can have commas in them (yes SSMS gets real support cases because of this). Make sure that cobra is passing correctly delimited strings and that this code is aware of whether the comma is part of the path or is the real separator between path and name